Placeeo/Model/GooglePlacesManager.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  GooglePlacesManager.swift
+//  Placeeo
+//
+//  Created by Pasha Berger on 11.10.2021.
+//
+
+import Foundation
+import GooglePlaces
+import CoreLocation
+final class GooglePlacesManager {
+    
+    let apiKey = "AIzaSyBLVcVo5N4mcs1PASpBMYzgpKuff_gkz8A"
+    
+    static let shared = GooglePlacesManager()
+    
+    private let client = GMSPlacesClient.shared()
+    
+    private init (){}
+    
+    enum PlacesError: Error{
+        case failedToFind
+        case failedToGetCoords
+    }
+    
+    func getPlaces(around location: CLLocationCoordinate2D, completion: @escaping ([Place]) -> Void) {
+        guard let url = URL(string: "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=\(location.latitude),\(location.longitude)&radius=50000&type=restaurant&key=\(apiKey)") else {
+            completion([])
+            return
+        }
+        
+        URLSession.shared.dataTask(with: url) { data, response, error in
+            guard let data = data else {return}
```

Comment:
don't forget to follow the common code style:
```suggestion
            guard let data = data else { return }
```
please check other places as well. I noticed plenty of such violations